automated-comments/typescript/general/prefer_templated_strings.md

```diff
@@ -0,0 +1,7 @@
+Use a [templated string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
```

Comment:
Actually, let's just use `template literal`, because that's what they're called in JavaScript.

Perhaps `String template literal`, to make it extra clear?

Reply:
I like it.